backend/src/modules/teams/services/get.team.service.ts

```diff
@@ -90,6 +87,27 @@ export default class GetTeamService implements GetTeamServiceInterface {
 		});
 	}
 
+	async getUsersOnlyWithTeams() {
+		const teams: LeanDocument<UsersWithTeams>[] = await this.teamUserModel.aggregate([
+			{
+				$lookup: {
+					from: 'teams',
+					localField: 'team',
+					foreignField: '_id',
+					as: 'teams'
+				}
+			},
+			{
+				$group: {
+					_id: '$user',
+					teamsNames: { $push: '$teams.name' }
+				}
+			}
+		]);
```

Comment:
```suggestion
	const teams: LeanDocument<UserWithTeams>[] = await this.teamUserModel.aggregate([
			{
				$lookup: {
					from: 'teams',
					localField: 'team',
					foreignField: '_id',
					as: 'teams'
				}
			},
			{
				$lookup: {
					from: 'users',
					localField: 'user',
					foreignField: '_id',
					as: 'user'
				}
			},
			{
				$unwind: { path: '$teams' }
			},
			{
				$unwind: { path: '$user' }
			},
			{
				$group: {
					_id: '$user',
					teamsNames: { $push: '$teams.name' }
				}
			},
			{
				$set: { user: '$_id' }
			},
			{
				$unset: [
					'_id',
					'user.currentHashedRefreshToken',
					'user.isSAdmin',
					'user.joinedAt',
					'user.isDeleted'
				]
			}
		]);
```

With this query it will return an object `[{user: userData, teams: [string]}]` with the user populated